docs/index.html

```diff
@@ -37,6 +37,10 @@ <h4>Include scripts</h4>
               <p>&lt;script type="text/javascript" src="chartjs-plugin-style.js"&gt;&lt;/script&gt;</p>
             </div>
           </div>
+          <p>Or with ES6 module import/<p>
+          <div class="shell">
+            <p>import 'chartjs-plugin-style';</p>
+          </div>
```

Comment:
`/<p>` means `</p>`?

Also, can you use `<div class="terminal">` and `div class="header">` tags like the previous lines?

Reply:
done